lib/iris/fileformats/pp.py

```diff
@@ -2414,6 +2422,11 @@ def save_pairs_from_cube(cube, field_coords=None, target=None):
         pp_field.lbcode = 1     # Grid code.
         pp_field.bmks = 1.0     # Some scaley thing.
         pp_field.lbproc = 0
+        # Set default MDI value.
+        # Note: *we no longer respect any fill_value of masked data arrays*.
```

Comment:
I will remove the historical reference !

I am concerned though that the effect of the new approach still needs be made a bit clearer :
The user should be able to see what fill-value is used + how you might control it
The developer should also be able to follow the implementation.